src/components/scenes/SendScene.js

```diff
@@ -279,7 +273,8 @@ class SendComponent extends React.PureComponent<Props, State> {
   }
 
   submit = async () => {
-    const { guiMakeSpendInfo, updateSpendPending, signBroadcastAndSave } = this.props
+    const { guiMakeSpendInfo } = this.props.route.params
+    const { updateSpendPending, signBroadcastAndSave } = this.props
```

Comment:
`const { updateSpendPending, signBroadcastAndSave, route } = this.props`, and then the next line can be `const { guiMakeSpendInfo } = route.params`